src/bin/rustfmt.rs

```diff
@@ -20,14 +20,16 @@ extern crate getopts;
 use rustfmt::{run, Input};
 use rustfmt::config::{Config, WriteMode};
 
-use std::env;
+use std::{env, error};
 use std::fs::{self, File};
 use std::io::{self, ErrorKind, Read, Write};
 use std::path::{Path, PathBuf};
 use std::str::FromStr;
 
 use getopts::{Matches, Options};
 
+type Error = Box<error::Error + Send + Sync>;
+type Result<T> = std::result::Result<T, Error>;
```

Comment:
Can you call this FmtResult rather than overloading Result (I know namespacing should address this, but Result being in the prelude makes this kind of annoying to read).